includes/RestApi/IntegrationsController.php

```diff
@@ -53,6 +54,17 @@ public function register_routes() {
 				),
 			)
 		);
+		\register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/hiive',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_hiive_token' ),
```

Comment:
Where are we using this API? I'm uneasy about passing a Hiive token to the front end.

Reply:
this endpoint we're using it in facebook module to store ``facebook token`` against ``hiive token``  as key-value pair in cloudflare